tests/test_wikipedia_dobule_search.py

```diff
@@ -0,0 +1,23 @@
+import time
+import allure
+import pytest
+from selenium.webdriver import Keys
+from selenium.webdriver.chrome import webdriver
+
+
+@allure.title("Search from phrase and verify if phrase is correct")
+def test_search_from_main_page(pages):
+    with allure.step("Search for first phrase"):
+        pages.wiki_home.search_bar.send_keys("Python")
+        with allure.step("click enter"):
+            pages.wiki_home.search_bar.send_keys(Keys.ENTER)
+        time.sleep(5)
```

Comment:
It's very tempting to use sleep, isn't it? :D As I wrote before, let's try to avoid it :)